File: Kotlin + JVM + Micronaut/src/main/kotlin/ac/lsys/model/Event.kt
package ac.lsys.model

import io.micronaut.data.annotation.GeneratedValue
import io.micronaut.data.annotation.Id
import io.micronaut.data.annotation.Index
import io.micronaut.data.annotation.MappedEntity
import io.micronaut.serde.annotation.Serdeable


@Serdeable
@MappedEntity
@Index(name = "idx_aggregate", columns = ["aggregateUUID"], unique = false)
data class Event(
    @field:Id
    @field:GeneratedValue(GeneratedValue.Type.AUTO)
    var id: Int = 0,
    var aggregateUUID: String,
    var data: String
)

File: JavaScript + Node.js + Express/src/entities/event.ts
// src/event.ts
import { Entity, PrimaryGeneratedColumn, Column, Index } from 'typeorm';

@Entity()
export class Event {
  @PrimaryGeneratedColumn()
  id!: number;

  @Index()
  @Column()
  aggregateUuid!: string;

  @Column()
  data!: string;
}